vhdbuilder/scripts/automate_release.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -euxo pipefail
+source vhdbuilder/scripts/automate_helpers.sh
+
+EV2_ARTIFACT_PIPELINE_ID="319030" 
```

Comment:
can we add a URL of the matching pipeline in ADO, for easier lookup

Reply:
im actually not sure adding a URL here makes sense since it's ADO-internal, and this is technically OSS...

Reply:
good call. I keep forgetting this is Public...